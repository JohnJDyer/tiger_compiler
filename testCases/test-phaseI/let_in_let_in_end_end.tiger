let var a: int; in let var b: int; in a:=b; end end