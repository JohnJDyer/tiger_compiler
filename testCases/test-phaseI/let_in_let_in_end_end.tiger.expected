main:
	assign, a, b, 
	return, , , 
